### PR TITLE
Minor refactoring of uxperf workloads

### PR DESCRIPTION
**Pull request by johnrchrdsn**
Delete saved images during teardown of googlephotos workload.  Refactor
selectPhoto helper method in googlephotos and add pause in Multiapp
workload for stability. Includes minor fixes.

### CODE REVIEW

File: wlauto/workloads/multiapp/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java [hunk 1]
```diff
@@ -90,6 +91,8 @@ private void logIntoSkype()  throws Exception {
         shareUsingApp("Skype");
         skype.handleLoginScreen(loginName, loginPass);
         confirmAccess();
+
+        sleep(10); // Pause while the app settles before returning
```
Comment (jimboatarm) [inline wlauto/workloads/multiapp/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java:95]: wow.. Long time.  Does it need to be this long?

File: wlauto/workloads/skype/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java [hunk 2]
```diff
@@ -48,7 +49,7 @@
 
     public void runUiAutomation() throws Exception {
         // Override superclass value
-        this.waitTimeout = 10000;
+        this.waitTimeout = TimeUnit.SECONDS.toMillis(10);
```
Comment (jimboatarm) [inline wlauto/workloads/skype/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java:52]: And again, long time

File: wlauto/workloads/multiapp/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java (hunk 1 above)
Comment (johnrchrdsn) [inline wlauto/workloads/multiapp/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java:95]: In reality it does't appear to take 10 seconds as other processes are in action while this sleep occurs. It was reliable at 10 seconds. I can test with smaller numbers but all this testing takes time. Maybe 30 minutes per change.

File: wlauto/workloads/skype/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java (hunk 2 above)
Comment (johnrchrdsn) [inline wlauto/workloads/skype/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java:52]: I did't set this. But I believe this was done for stability.

**Pull request merged**